bip-XXXX.mediawiki

```diff
@@ -415,18 +416,21 @@ The first part of the table shows how spam is reduced. The part where total sum
 
 ==Backward compatibility==
 
-Soft fork: If applied AND old hardcoded block size limit is kept.
+Soft fork: If applied AND old hardcoded block size limit is kept. (Note: This BIP will most probably not establish itself as softfork among miners, because their money driven incentive is to create a max sized block instead a smaller one.)
```

Comment:
1. Miners don't have a say in softforks.
2. This isn't true.

Reply:
>     1. Miners don't have a say in softforks.
> 
>     2. This isn't true.

Disagree.